src/components/accordion/accordion.style.ts

```diff
@@ -130,8 +131,19 @@ interface StyledAccordionTitleContainerProps {
   size?: "large" | "small";
 }
 
+const oldFocusStyling = `
+  outline: solid 3px var(--colorsSemanticFocus500);
```

Comment:
This change was done before I came onto the PR, not sure if there was a legitimate reason for it being changed. Personally, I'm not hugely concerned since the value would equal 3px and as mentioned the theme of this PR will mean that we don't care too much about the oldFocusStyling beyond providing existing support. There's an argument to say that if it's only for legacy projects we shouldn't even use the design tokens at all but that's a different battle. 

However, if yourself and others think we should put the design token in, I'm of course happy to do that, I'm just trying to avoid unnecessary changes to this already lengthy and time critical PR where possible.